patches/expo-modules-core+1.11.8+002+disableViewRecycling.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/node_modules/expo-modules-core/ios/Fabric/ExpoFabricViewObjC.mm b/node_modules/expo-modules-core/ios/Fabric/ExpoFabricViewObjC.mm
+index 96bba7d..6e0b439 100644
+--- a/node_modules/expo-modules-core/ios/Fabric/ExpoFabricViewObjC.mm
++++ b/node_modules/expo-modules-core/ios/Fabric/ExpoFabricViewObjC.mm
+@@ -155,6 +155,11 @@ - (void)updateProps:(const facebook::react::Props::Shared &)props oldProps:(cons
+   [self viewDidUpdateProps];
+ }
+ 
+++ (BOOL)shouldBeRecycled
```

Comment:
same here, don't see an upstream PR. Not sure the context here or if this change would make sense upstream